### PR TITLE
Update app-sdk-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Appstore Submission Note

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review.

**Comment by PRMerger3** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Erikre

File: memdocs/intune/developer/app-sdk-ios.md [hunk 1]
```diff
@@ -80,6 +80,9 @@ To enable the Intune App SDK, follow these steps:
 
     :::image type="content" source="media/app-sdk-ios/intune-app-sdk-ios-linked-framework.png" alt-text="Intune App SDK iOS Framework: Xcode Frameworks, Libraries, and Embedded Content sample":::
 
+   > [!NOTE]
+   > If you use the framework, you must manually strip out the simulator architectures from the universal framework before you submit your app to the App Store. See [Submit your app to the App Store](#submit-your-app-to-the-app-store) for more details.
```
Comment (JohanFreelancer9) [inline memdocs/intune/developer/app-sdk-ios.md:84]:
```suggestion
   > If you use Framework, you must manually strip out the simulator architectures from the universal framework before you submit your app to the App Store. See [Submit your app to the App Store](#submit-your-app-to-the-app-store) for more details.
```

File: memdocs/intune/developer/app-sdk-ios.md [hunk 2]
```diff
@@ -1046,3 +1049,22 @@ Yes! Please see the [Chatr sample app](https://github.com/msintuneappsdk/Chatr-S
 ### How can I troubleshoot my app?
 
 The Intune SDK for iOS 9.0.3+ supports the ability to add a diagnostics console within the mobile app for testing policies and logging errors. `IntuneMAMDiagnosticConsole.h` defines the `IntuneMAMDiagnosticConsole` class interface, which developers can use to display the Intune diagnostic console. This allows end users or developers during test to collect and share Intune logs to help diagnose any issue they may have. This API is optional for integrators.
+
+
+## Submit your app to the App Store
+
+Both the static library and framework builds of the Intune App SDK are universal binaries. This means they have code for all device and simulator architectures. Apple will reject apps submitted to the App Store if they have simulator code. When compiling against the static library for device-only builds, the linker will automatically strip out the simulator code. Follow the steps below to ensure all simulator code is removed before you upload your app to the App Store.
```
Comment (JohanFreelancer9) [inline memdocs/intune/developer/app-sdk-ios.md:1056]:
```suggestion
Both the static library and Framework builds of the Intune App SDK are universal binaries. This means they have code for all device and simulator architectures. Apple will reject apps submitted to the App Store if they have simulator code. When compiling against the static library for device-only builds, the linker will automatically strip out the simulator code. Follow these steps to ensure all simulator code is removed before you upload your app to the App Store:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, please commit these changes. Many thanks.

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and signoff this PR. Thanks

**Comment by yogkumgit** [comment]
@Erikre Gentle reminder

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: memdocs/intune/developer/app-sdk-ios.md [hunk 3]
```diff
@@ -80,6 +80,9 @@ To enable the Intune App SDK, follow these steps:
 
     :::image type="content" source="media/app-sdk-ios/intune-app-sdk-ios-linked-framework.png" alt-text="Intune App SDK iOS Framework: Xcode Frameworks, Libraries, and Embedded Content sample":::
 
+   > [!NOTE]
+   > If you use Framework, you must manually strip out the simulator architectures from the universal framework before you submit your app to the App Store. See [Submit your app to the App Store](#submit-your-app-to-the-app-store) for more details.
```
Comment (tiburd) [inline memdocs/intune/developer/app-sdk-ios.md:84]:
```suggestion
   > If you use Framework, you must manually strip out the simulator architectures from the universal framework before you submit your app to the App Store. For more information, see [Submit your app to the App Store](#submit-your-app-to-the-app-store).
```

**Pull request merged**